website/content/docs/connect/ca/vault.mdx

```diff
@@ -279,6 +317,137 @@ path "/connect_inter/*" {
 
 </CodeBlockConfig>
 
+## Configuration with Vault Namespaces
+
+[Vault Namespaces](https://www.vaultproject.io/docs/enterprise/namespaces)
+are a multi-tenancy feature of Vault Enterprise that allow
+multiple, isolated Vault environments within a single Vault infrastructure.
+
+This Connect CA provider can access Vault tokens, auth methods, and PKI secrets engines
+that exist within the same or different Vault Namespaces.
+
+The [`Namespace`](#namespace) option is used as the `X-Vault-Namespace` header
+for all API requests made by this CA provider to Vault.
+All Vault namespaces included in paths are relative to the value of this header.
+For more details on how paths to namespaced resources can be specified,
+refer to the [Vault Namespace usage documentation](https://www.vaultproject.io/docs/enterprise/namespaces#usage).
+
+The PKI path options ([`RootPKIPath`](#rootpkipath) and [`IntermediatePKIPath`](#intermediatepkipath)
+both support including a Vault Namespace in the path.
+However, the Vault authentication options ([`Token`](#token) and [`AuthMethod`](#authmethod))
```

Comment:
@markan : my conclusion from reading the [auth method code](https://github.com/hashicorp/consul/pull/11573/files#diff-7bf30611a760296e2e6ffcd7fe955b0eb8317fe0a43c470b8b52d311af4a2515R744) is that namespaces cannot be provided in the path for that configuration. `AuthMethod.MountPath` seems to be injected to API calls with:
```
loginPath = fmt.Sprintf("auth/%s/login", authMethod.MountPath)
```

... when the Vault Namespace would have to come before `auth/`. So, for both Vault tokens and auth methods, the namespace must be provided in the `Namespace` option rather than in the path. I tried to document that clearly in this section.

Reply:
You are correct. Also we should note that the Namespace field doesn't affect this in 1.11.0-1.11.5 because of a bug. This is has been corrected in 1.12.0 and backported to the 1.11.x branch (https://github.com/hashicorp/consul/pull/12793). No change is needed, noting this mostly for posterity.

Reply:
I'll probably include this caveat in the 1.11 version of the docs (backport to that).

I need to rewrite these docs anyways given what we're learned / the config UX enhancements we're in the process of making in 1.12.x for Vault namespaces.